io/net/bootlist_test.py.data/README.txt

```diff
@@ -0,0 +1,13 @@
+Testing the bootlist for Normal,Service and Both mode.
+Here we converting to all the mode and displaying the
+output as logical device name and open firmware devices.
```

Comment:
can this re worded , as below 
Test does testing bootlist for Normal and Service mode even Both mode ,as converting to all modes ad display ouiput as logical Device name and open firmware devices.